include/actionlib/managed_list.h

```diff
@@ -154,12 +154,16 @@ class ManagedList
       T& getElem()
       {
         assert(valid_);
+        if (!valid_) 
+          ROS_ERROR_NAMED("actionlib", "getElem() should not see non-valid handles");
```

Comment:
Please use `{ ... }`. Otherwise the logic is broken if the macro is being defined as empty.

Arguably raising an exception might be more appropriate here.

Same below.

Nitpick: `non-valid` -> `invalid`.